modules/__tests__/createPrefixer-test.js

```diff
@@ -431,5 +431,119 @@ describe('Static Prefixer', () => {
         expect(prefix(input)).toEqual(output)
       })
     })
+
+    describe('grid', () => {
+      it('should prefix display values', () => {
+        const input = { display: 'grid' }
+        const inlineInput = { display: 'inline-grid' }
+        const output = { display: ['-ms-grid', 'grid'] }
+        const inlineOutput = { display: ['-ms-inline-grid', 'inline-grid'] }
+        expect(prefix(input)).toEqual(output)
+        expect(prefix(inlineInput)).toEqual(inlineOutput)
+      })
+
+      it('should prefix basic alignment values', () => {
+        const input = { alignSelf: 'stretch', justifySelf: 'end' }
+        const output = {
+          WebkitAlignSelf: 'stretch',
+          alignSelf: 'stretch',
+          justifySelf: 'end',
+          msFlexItemAlign: 'stretch',
+          msGridColumnAlign: 'end',
+          msGridRowAlign: 'stretch',
+        }
+        expect(prefix(input)).toEqual(output)
+      })
+
+      describe('template', () => {
+        it('should prefix column templating', () => {
+          const input = { gridTemplateColumns: '1fr auto' }
+          const output = {
+            gridTemplateColumns: '1fr auto',
+            msGridColumns: '1fr auto',
+            msGridTemplateColumns: '1fr auto', // Not a valid property
```

Comment:
The list of properties defined [here](https://github.com/rofrischmann/inline-style-prefixer/blob/master/modules/generator/maps/propertyMap.js#L126) causes this prefix to be automatically generated, even if it isn't valid (to the best of my knowledge). I don't have full context on why this is generated, but I think it could be removed - just waiting to hear back for confirmation.